### PR TITLE
Gene Expression integration bundle

### PR DESCRIPTION
**Pull request by caravinci**
## Environment to be used in testing

- [ ] `prod`
- [x] `dev-oicr`
- [x] `qa` (not sure which version)

## Description of Changes
See commits' descriptions.

### CODE REVIEW

File: src/packages/@ncigdc/utils/ajax/index.js [hunk 1]
```diff
@@ -35,11 +35,25 @@ export function fetchAuth(options: { endpoint: string }): Object {
 }
 
 // $FlowIgnore
-export const fetchApi = (endpoint, opts = {}) => {
+export const fetchApi = (endpoint, options = {}) => {
```
Comment (samrichca) [inline src/packages/@ncigdc/utils/ajax/index.js:38]: I think `opts` is missing

Comment (caravinci) [reply src/packages/@ncigdc/utils/ajax/index.js:38]: Hmm must've gotten lost somewhere along the way (merges etc.)

Comment (caravinci) [reply src/packages/@ncigdc/utils/ajax/index.js:38]: The opts come from line 47 instead ;)

Comment (samrichca) [reply src/packages/@ncigdc/utils/ajax/index.js:38]: oh I see what you did there

:+1:

**Comment by caravinci** [comment]
Pushed a develop-rebased version, rather than merging develop on top, to keep the timeline less polluted.

**Pull request merged**